ros2_batch_job/packaging.py

```diff
@@ -75,33 +67,33 @@ def build_and_test_and_package(args, job):
             '--build-base', '"%s"' % args.buildspace,
             '--install-base', '"%s"' % args.installspace,
             '--cmake-args', '-DBUILD_TESTING=ON', '--no-warn-unused-cli',
-            '--packages-select', 'ros1_bridge',
             '--event-handlers', 'console_direct+',
-        ] + (['--merge-install'] if not args.isolated else []) +
+        ] + (['--packages-select', overlay_pkgs] if overlay_pkgs else []) +
+            (['--merge-install'] if not args.isolated else []) +
             (
                 ['--cmake-args', '-DCMAKE_BUILD_TYPE=' +
                     args.cmake_build_type]
                 if args.cmake_build_type else []
         ), env=env)
         print('# END SUBSECTION')
 
-        if args.test_bridge:
-            print('# BEGIN SUBSECTION: install Python packages for ros1_bridge test')
-            job.run(['"%s"' % job.python, '-m', 'pip', 'install', '-U', 'catkin_pkg', 'rospkg'], shell=True)
-            print('# END SUBSECTION')
+        print('# BEGIN SUBSECTION: install Python packages for ros1_bridge test')
+        job.run(['"%s"' % job.python, '-m', 'pip', 'install', '-U', 'catkin_pkg', 'rospkg'], shell=True)
+        print('# END SUBSECTION')
```

Comment:
Should this block be further down in the condition `if args.mixed_ros_overlay_pkgs:`?